examples/specs/layer_bar_dual_axis_minmax.vl.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
This one is nice though because it is a layer of unit and layer. And it's actually meaningful.